lib/unlocked/ManageBoards.dart

```diff
@@ -0,0 +1,844 @@
+                                                          ),
+                                                          Text(
+                                                            'This action will create a copy of the public board and will add it into your communicator. Do you want to proceed?',
+                                                            style: TextStyle(
+                                                                fontSize: 16),
+                                                          ),
+                                                          SizedBox(
+                                                            height: 15,
+                                                          ),
+                                                          Row(
+                                                            mainAxisAlignment:
+                                                                MainAxisAlignment
+                                                                    .spaceAround,
+                                                            children: [
+                                                              RaisedButton(
+                                                                splashColor:
+                                                                    paua,
+                                                                disabledColor:
+                                                                    Colors
+                                                                        .white,
+                                                                shape: RoundedRectangleBorder(
+                                                                    side: BorderSide(
+                                                                        color:
+                                                                            paua),
+                                                                    borderRadius:
+                                                                        BorderRadius.circular(
+                                                                            20.0)),
+                                                                child: Text(
+                                                                  "No",
+                                                                  style: TextStyle(
+                                                                      fontSize:
+                                                                          16.0,
+                                                                      color:
+                                                                          paua),
+                                                                ),
+                                                              ),
+                                                              RaisedButton(
```

Comment:
Similarly, you may want to change this to `ElevatedButton`
Edited: we have `button` class in `shared` folder, you may want to use this.